### ISSUE TITLE
MCDU Track miles freezes on arrivals

### ISSUE THREAD

**Issue by jnmonks47** [flybywiresim/a32nx#5656]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-08-14T20:40:24+00:00",
  "ref": "master",
  "sha": "2019690439abe7de5735af4730ef51f1d723e8b6",
  "actor": "MisterChocker",
  "event_name": "manual"
}
```

### Describe the bug

On arrivals into airport, Distance on the MCDU (Track miles to-go) freeze and doesn't continue dropping when flying, leading to inaccurate distance to go. Has now happened on the last 3 flights

### Expected behavior

Distance to go displayed should continue to fall per mile flown

### Steps to reproduce

1. Set up a flight plan into an airport with arrival and approach procedure. 
2. Continue flying until distance freezes, then doesn't correct

### Discord Username (optional)

jnmonks47

**Comment by Mickg56** [comment]
concur

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
